src/components/content/resource/helpers/details-card.tsx

```diff
@@ -114,71 +121,45 @@ const transposeTable = (tbl: ReactElement): ReactElement => {
     return tbl;
   }
 
-  let transposeIdx = 0;
+  // let transposeIdx = 0;
   dataRows.forEach((dataRow, rowIdx) => {
-    // each data row should equate to N new tranpose rows, one for
-    // each column in the original data table
-    newRows.push(
-      ...headerRows.map((col, colIdx) => (
-        // the child should be just one text value
-        <tr key={`transposed-${transposeIdx}`}>
-          <td
-            key={
-              // eslint-disable-next-line react/no-array-index-key
-              `col-${colIdx}`
-            }
-          >
-            {col.props.children}
-          </td>
-        </tr>
-      ))
-    );
-
-    if (rowIdx > 0) {
-      // get the next tranposed row that aligns with the next data row
-      const tr = newRows[transposeIdx];
-      newRows[transposeIdx] = (
-        <tr key={tr.key} className="ctw-outline">
-          {tr.props.children}
-        </tr>
-      );
-    }
-
     // now for each cell in the data row
     const dataCells = Array.isArray(dataRow.props.children)
       ? (dataRow.props.children as ReactElement[])
       : ([dataRow.props.children] as ReactElement[]);
 
     dataCells.forEach((dataCell, cellIdx) => {
-      const tr = newRows[transposeIdx];
-      const currentCells = Array.isArray(tr.props.children)
-        ? (tr.props.children as ReactElement[])
-        : ([tr.props.children] as ReactElement[]);
-      const updatedCells = [
-        ...currentCells,
-        // eslint-disable-next-line react/no-array-index-key
-        <td key={`data-${transposeIdx}-${cellIdx}`}>{dataCell.props.children}</td>,
-      ];
+      const headerRow = headerRows[cellIdx];
 
-      newRows[transposeIdx] = (
-        <tr key={`transposed-${transposeIdx}`} className={tr.props.className}>
-          {updatedCells}
-        </tr>
+      newRows.push(
+        // eslint-disable-next-line react/no-array-index-key
+        <div key={`cell-${rowIdx}-${cellIdx}`}>
+          <p>
+            <b>{headerRow.props.children}</b>
+          </p>
```

Comment:
`<b>` should be something like  `.ctw-font-medium` or `.ctw-font-semibold`
```suggestion
          <p className="ctw-font-semibold">{headerRow.props.children}</p>
```